docs: Implement style guide rules - active voice, misc (#23272)
MystenLabs/sui: docs/content/concepts/object-ownership/shared.mdx

keywords: [ shared objects, sharing objects, objects that are shared, public objects, accessing shared objects, accessing public objects ]
---

> A shared object is created using the `sui::transfer::share_object` function, making it publicly accessible on the network. Extended functionality and accessibility of shared objects requires additional effort by securing access, if needed.

Reviewer: ```suggestion
Use the `sui::transfer::share_object` function to create a shared object, making it publicly accessible on the network. Extended functionality and accessibility of shared objects requires additional effort by securing access, if needed.
```
